src/bin/neptune-cli.rs

```diff
@@ -34,7 +34,7 @@ use neptune_cash::models::state::wallet::wallet_file::WalletFile;
 use neptune_cash::models::state::wallet::wallet_file::WalletFileContext;
 use neptune_cash::models::state::wallet::wallet_status::WalletStatus;
 use neptune_cash::models::state::wallet::wallet_status::WalletStatusExportFormat;
-use neptune_cash::rpc_auth;
+use neptune_cash::{display_banner, rpc_auth};
```

Comment:
We use "item" granularity for imports to avoid merge conflict headaches. Please un-merge these imports.